TrackingTools/TrajectoryFiltering/interface/SeedExtensionTrajectoryFilter.h

```diff
@@ -0,0 +1,51 @@
+#ifndef SeedExtensionTrajectoryFilter_H
+#define SeedExtensionTrajectoryFilter_H
+
+#include "TrackingTools/TrajectoryFiltering/interface/TrajectoryFilter.h"
+
+class SeedExtensionTrajectoryFilter final : public TrajectoryFilter {
+public:
+
+  explicit SeedExtensionTrajectoryFilter() {} 
+  
+  explicit SeedExtensionTrajectoryFilter(edm::ParameterSet const & pset, edm::ConsumesCollector&) :
+     theStrict(pset.existsAs<bool>("strictSeedExtension") ? pset.getParameter<bool>("strictSeedExtension"):false),
```

Comment:
what prevents making fillDescriptions for these?
Use of pset.existsAs in new code is an exception that needs to be justified.